iot_config/src/route_service.rs

```diff
@@ -677,8 +895,8 @@ struct DevAddrEuiValidator {
 
 #[derive(thiserror::Error, Debug)]
 enum DevAddrEuiValidationError {
-    #[error("devaddr range outside of constraint bounds {0}")]
-    RangeOutOfBounds(String),
+    #[error("devaddr or range outside of bounds {0}")]
```

Comment:
From what I can follow, this error is only used when a Route's Devaddr Range falls outside the Org's constraints. 
So I _think_ this might be a typo?

Reply:
not a typo, just a holdover. when i started out i was going unify all of these under a single update validator but i switched to handling the skf updates differently since they aren't being streamed; i just forgot to set this error message back to what it had originally been. good catch